cmd/minishift/cmd/root_test.go

```diff
@@ -112,9 +113,9 @@ func TestPreRunDirectories(t *testing.T) {
 func TestViperConfig(t *testing.T) {
 	defer viper.Reset()
 	err := cli.InitTestConfig(`{ "v": "999" }`)
-	if viper.GetString("v") != "999" || err != nil {
-		t.Fatalf("Viper cannot read the test config file: %v", err)
-	}
+
+	assert.NoError(t, err)
+	assert.Equal(t, viper.GetString("v"), "999")
```

Comment:
`assert.Equal(t, viper.GetString("v"), "999", "Viper cannot read the test config file")`